src/edu/csus/ecs/pc2/clics/API202003/ClarificationService.java

```diff
@@ -21,11 +21,10 @@
 import edu.csus.ecs.pc2.core.model.Clarification;
 import edu.csus.ecs.pc2.core.model.ClarificationAnswer;
 import edu.csus.ecs.pc2.core.model.IInternalContest;
-import edu.csus.ecs.pc2.core.util.JSONTool;
 
 /**
  * WebService to handle languages
```

Comment:
This does not appear to be a "WebService to handle LANGUAGES"...

Reply:
Same - old code base.

Reply:
Same reply :)